chroma-manager/setup.py

```diff
@@ -28,7 +28,9 @@
     # file creation/cleanup in the Makefile) to deal with the fact
     # that setuptools wants to strip the first character off the filename.
     package_data = {
-        '': [".chroma-manager.py", ".production_supervisord.conf", ".chroma-manager.conf.template", ".mime.types"],
+        '': [".chroma-manager.py", ".production_supervisord.conf",
```

Comment:
almost not worth commenting , but looks strange with different number of elements on each line in list